pennylane/ops/op_math/pow.py

```diff
@@ -322,6 +345,21 @@ def diagonalizing_gates(self):
         """
         return self.base.diagonalizing_gates()
 
+    @property
+    def _pauli_rep(self):
+        """PauliSentence representation of the power of operations."""
+        if isinstance(self.z, int) and self.z > 0:
+            try:
+                pr = qml.pauli.PauliSentence({})
+                for _ in range(self.z):
+                    pr = pr * self.base._pauli_rep  # pylint: disable=protected-access
```

Comment:
I wonder if it's worth adding an `__pow__` method to PauliWord/ PauliSentence? Especially since a pauli word squared is always going to be the identity.  We want to be able to take advantage of that fact.